qiskit/circuit/library/n_local/excitation_preserving.py

```diff
@@ -116,8 +116,7 @@ def excitation_preserving(
     theta = Parameter("θ")
     if num_qubits > 1:
         swap = QuantumCircuit(2, name="Interaction")
-        swap.rxx(theta, 0, 1)
-        swap.ryy(theta, 0, 1)
+        swap.append(XXPlusYYGate(2 * theta), [0, 1])
```

Comment:
The factor of 2 is ugly but I added it to maintain equivalence with the previous code.